test/Faker/Core/MedicalTest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Faker\Test\Core;
+
+use Faker\Test\TestCase;
+
+final class MedicalTest extends TestCase
+{
+    public function testBloodType(): void
+    {
+        self::assertContains($this->faker->bloodType(), ['A', 'AB', 'B', 'O']);
```

Comment:
Can we use a more appropriate assertion?

As far as I am concerned, when using `assertContains()`, then `$needle` would be the expected value here, whereas `$haystack` is the actual value.

Right now it is the other way around.

Reply:
The needle is always the value and the haystack the expected? In this case needle also needs to be a string.

This is a 1:1 copy from the existing tests so i wonder where you want to?